service/worker/archiver/client.go

```diff
@@ -92,7 +93,7 @@ func (c *client) Archive(request *ArchiveRequest) error {
 			logging.TagWorkflowExecutionID: exec.ID,
 			logging.TagWorkflowRunID:       exec.RunID,
 		}).Error("failed to SignalWithStartWorkflow to archival system workflow")
-		c.metricsClient.IncCounter(metrics.ArchiverClientScope, metrics.ArchiverClientSendSignalFailureCount)
```

Comment:
It seems like ArchiverClientSendSignalFailureCount is only used here. If it's not used, shall we delete it from metric/def.go?

Reply:
nice catch